rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,507 @@
+will do the majority of project management and execution for the
+following deliverables. They will coordinate and collaborate with DCP
+partners ("contact person") to validate design correctness and minimize
+code changes and their impact.
+
+| Component/Deliverable                                 	| Responsible Team/Service 	| No earlier than 	| Contact Person	|
+|-------------------------------------------------------	|--------------------------	|-----------------	|---------------	|
+| HCA Metadata Draft                                    	| EBI                      	| Complete        	| Z. Perova     	|
+| Test Datasets Available                               	| starfish                 	| Complete        	| A. Carr       	|
+| Submission Datasets Available                         	| starfish                 	| Q2 2019         	| A. Carr       	|
+| Analysis & Adapter Pipelines                          	| Broad                    	| Q2 2019         	| A. Carr       	|
+| Lira subscription                                     	| Broad                    	| ?             	| S. Ehsan      	|
+| Data Processing Pipelines Tests                        	| Broad                    	| Q2 2019         	| G. Grant      	|
+| Specification for input format                        	| starfish                 	| Complete        	| J. Moore      	|
+| Specification for output format                       	| starfish                 	| Q1 2019         	| A. Carr       	|
+| Input validation tool                                 	| starfish, OME            	| Q3 2019         	| J. Moore      	|
+| Output validation tool                                	| starfish, OME            	| Q2 2019         	| A. Carr       	|
+| Ingest support for the nested arrays of channels      	| EBI                      	| ?               	| Z. Perova           |
+| [Upload and download of subdirectories][b]             	| ?                        	| ?               	| H. Schmidt           |
+| DSS spatial data bundle (or bundless) specification   	| UCSC                     	| ?               	| B. Hannafious 	|
+| DSS scale testing                                     	| UCSC / DCP               	| ?             	| B. Hannafious 	|
+| Data Browser facets for spatial data                  	| UCSC                     	| ?             	| H. Schmidt    	|
+| Matrix service integration                            	| starfish                 	| ?             	| M. Kinsella   	|
+| Upload Bundle for 1M+ files                           	| ?                        	| ?               	| P. Shah       	|
+| Image file download latency testing                   	| ?                        	| ?               	| B. Hannafious             	|
+
+### **Unresolved Questions**
+
+  - Who are the intended beta users for imaging support and what
+    functionality is being targeted?
+
+  - Clearly define a wrangling process for imaging data.
```

Comment:
Seems like wrangling process could be added to the table above?

Reply:
@brianraymor @ambrosejcarr I suggest adding "Transformation of data into the accepted format" rather than wrangling process itself. We have a defined wrangling process, however, the imaging data requires transformation into the SpaceTx format in order to be wrangled. The wranglers do not have the capacity to do it, and last week's SpaceTx Hackathon showed that it is highly dependent on the data and requires a good understanding of both the pipeline and the experimental approach. Therefore I think it is crucial to determine who will be responsible for this. I see two options: to find a more simple way to transform the data and expect data contributors to do it themselves, or allocate resources for a dedicated person to do it for all the data coming into HCA.

Reply:
Added.